pilot/pkg/xds/delta.go

```diff
@@ -79,6 +74,11 @@ func (s *DiscoveryServer) StreamDeltas(stream DeltaDiscoveryStream) error {
 		deltaLog.Debugf("Unauthenticated XDS: %v", peerAddr)
 	}
 
+	if err := s.WaitForRequestLimit(stream.Context()); err != nil {
```

Comment:
revert

Reply:
I did revert it. Looks like file not saved. reverted now